frontend/src/main/components/Users/UsersTable.js

```diff
@@ -1,7 +1,8 @@
 
 import OurTable, { ButtonColumn } from "main/components/OurTable"
 import { useBackendMutation } from "main/utils/useBackend";
-
+//import UsersTable from './UsersTable';
```

Comment:
Please do not submit (or approve with LGTM) PRs for review that have commented out code.

Reply:
removed